core_contracts/daofund/daofund.py

```diff
@@ -199,13 +211,11 @@ def tokenFallback(self, _from: Address, _value: int, _data: bytes) -> None:
         address = str(self.msg.sender)
         for symbol in assets:
```

Comment:
Seems like we do not really need to reference Loans at all, right? Seems line only lines 211, 216, 217, and 218 are needed.